Troubleshooting.md

```diff
@@ -0,0 +1,33 @@
+# Troubleshooting
+If you have problems with the La Capitaine Icon Theme, please read the following sections before submitting an issue on GitHub. `.desktop` files usually can be found in `/usr/share/applications/` (accessible for all users) or in `~/.local/share/applications/` (accessible for the current user). To see if some application icon is available in La Capitaine, just browse `apps/scalable/` in this repository and have a look. If you got lucky, the icon can be applied to any application by placing `Icon=cool-icon-name` in the application's `.desktop` file, assuming `apps/scalable/cool-icon-name.svg` exists.
```

Comment:
rather than `cool-icon-name`, it would be better to use programmer jargon, such as `foo-icon-name`